images/manageiq-base/Dockerfile

```diff
@@ -45,18 +47,43 @@ RUN yum -y install https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-re
                    lvm2             \
                    net-tools        \
                    nmap-ncat        \
-                   nodejs           \
                    openldap-clients \
                    openscap-scanner \
                    patch            \
                    psmisc           \
-                   postgresql-devel \
                    sqlite-devel     \
                    sysvinit-tools   \
                    which            \
+		   wget 	    \
+		   make		    \
+		   readline-devel   \
+		   zlib-devel	    \
+		   perl-devel	    \
+		   bison	    \
+		   flex		    \
                    &&               \
     yum clean all
 
+RUN wget  https://github.com/postgres/postgres/archive/REL_10_11.tar.gz && \
+    tar -zxvf REL_10_11.tar.gz && cd postgres-REL_10_11/ && ./configure --prefix=/usr && make && make install && \
```

Comment:
Would really prefer this rpm to be built outside of the Dockerfile.

We use copr for this. @simaishi would it be difficult to rebuild the rpms we host in copr for the different arch? Or should @seth-priya create a separate copr account for this?

Reply:
I think we should add another arch to the existing ones. I'll give it a try.

Reply:
@seth-priya I built postgresql rpm. I think that's the only rpm needed. Please test the rpm using https://copr.fedorainfracloud.org/coprs/simaishi/test/repo/epel-7/simaishi-test-epel-7.repo

If it looks good, I'll make the RPM available in manageiq/ManageIQ-Master.

Reply:
thanks @simaishi I will test the RPM

Reply:
@simaishi I have tested the postgresql RPM and seems to work perfectly, I will update the PR with my changes, once you publish it to the main repo, thank you!!!

Reply:
In manageiq/ManageIQ-Master now. Thanks!